README.rst

```diff
@@ -17,7 +17,7 @@ Use pip to install!
 
     pip install keen
 
-This client is known to work on Python 2.6, 2.7, 3.2, 3.3, 3.4 and 3.5.
+This client is known to work on Python 2.6, 2.7, 3.2, 3.3, 3.4, 3.5 and 3.6.
```

Comment:
Issue #126 is the reason. I don't consider Python 3.2 "not supported" anymore...yet. I really just don't know right now what started happening. If the problem is that pycryptodome 3.4.5 magically stopped supporting 3.2 (doubtful, since that would mean they pushed a breaking change without updating version numbers) then that's more serious. I kinda hope it's just a Travis image problem that will get fixed and we can Python 3.2 back to the Travis CI list of environments.